package.json

```diff
@@ -73,7 +73,6 @@
         "test:watch": "yarn test:watch:focus unit",
         "test:watch:flags:focus": "yarn build && run-p build:watch \"test:start --watch --group {1} --config web-test-runner.config.ci-chromium-flags.js\" --",
         "test:watch:focus": "yarn build && run-p build:watch \"test:start --watch --group {1}\" --",
-        "verify-build-artifacts": "node ./scripts/verify-build-artifacts.js",
```

Comment:
A couple question - are we sure we want to delete this script? It does validate the build before release which is a rather nice sanity check. Not necessary perhaps but good for peace of mind. If you do think we should remove it, can you delete it in this PR too from the folder?

Reply:
verify-build-artifacts didnt exist in the scripts folder but maybe we need to add it in and add this command back? it was removed in the scripts work but could have been my oversight